chat-ui/vitest.config.ts

```diff
@@ -0,0 +1,38 @@
+import { defineConfig, mergeConfig } from "vitest/config";
+import viteConfig from "./vite.config.component";
+
+const testConfig = defineConfig({
+  test: {
+    environment: "jsdom",
+    globals: true,
+    // alias: [
+    //   {
+    //     find: new RegExp("^@lg-chat/message$", "g"),
+    //     replacement: "@lg-chat/message/dist/index.js",
+    //   },
+    // ],
```

Comment:
What's this for?

Reply:
Leftover debugging of ESM issues when testing the components. `react-markdown` (used in `Message`) causes ESM import issues in test so this was trying to work around it.